cdap-docs/cdap-apps/source/hydrator/custom.rst

```diff
@@ -450,18 +453,30 @@ Plugin Deployment
    :start-after: .. _plugins-deployment-artifact:
    :end-before:  .. _plugins-use-case:
 
-Configuration JSON for a plugin (for Webapp UI alone)
-======================================================
-Every property of a plugin is represented, by default, as input fields in the UI. This can be however changed to specific widgets that closely match the type of the property in a plugin. This can be specified as JSON for the UI to consume.
+.. _cdap-apps-custom-plugin-configuration-json:
+
+Plugin Configuration JSON
+-------------------------
+When a plugin is displayed in the CDAP UI, the plugin and its properties are represented
+by widgets in the :ref:`Cask Hydrator <cdap-apps-cask-hydrator>`. Each property of a
+plugin is represented, by default, as an input field in the user interface of a widget. 
+
+A plugin can include a configuration JSON file that specifies a particular widget and set
```

Comment:
Isn't this line and line 467 kind of contradicting? We say it specifies a particular widget and set properties (which is not true. We don't set propertis) but at line 467 we say it is composed of two list. I am a little confused on reading both these statements.